src/mutation_events.js

```diff
@@ -214,4 +214,9 @@
   };
 
   console.log(`Mutation Events polyfill installed (native feature: ${nativeFeatureSupported ? "supported" : "not present"}).`);
+
+  // Helper function to avoid errors when has doesn't exist
```

Comment:
Mind moving this up to around line 129 or so?

Reply:
Done